framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Logo/Logo.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/constants';
+
+.wrapper {
+	display: inline-flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	height: 53px;
+	color: $color-white;
```

Comment:
with line height added to text, you can `align-items: center` should be here?

Reply:
With current implementation its much stable to use, in case font is not loaded from web and line-height is not set properly. The flex current options will still make them aligned to the logo.